packages/block-library/src/navigation-link/block.json

```diff
@@ -44,6 +44,9 @@
 		},
 		"isTopLevelLink": {
 			"type": "boolean"
+		},
+		"metadata": {
+			"type": "object"
 		}
```

Comment:
@talldan I wonder if we need this? Is it auto-added when you initialise a block binding? 🤔

Reply:
Thanks. They are not required.